includes/admin/class-support-admin.php

```diff
@@ -549,6 +577,52 @@ public function cloudflare_token_element_callback( $args ) {
 		include dirname( __FILE__ ) . '/views/html-cloudflare-token-field.php';
 	}
 
+	/**
+	 * NerdPress Relay Server URL Callback
+	 *
+	 * @param array $args Callback arguments.
+	 */
+	public function nerdpress_relay_server_url_element_callback( $args ) {
+		$menu    = $args['menu'];
+		$id      = $args['id'];
+		$options = get_option( $menu );
+		$token   = '';
+
+		if ( isset( $options['np_relay_api_token'] ) ) {
+			$url = $options['np_relay_api_token'];
+		} else {
+			$url                       = '';
+			$options['np_relay_api_token'] = $url;
+			update_option( 'blog_tutor_support_settings', $options );
+		}
+
+		include dirname( __FILE__ ) . '/views/html-nerdpress-relay-server-url-field.php';
+	}
+
+	/**
+	 * NerdPress Relay Server URL Callback
+	 *
+	 * @param array $args Callback arguments.
+	 */
+	public function nerdpress_relay_api_token_element_callback( $args ) {
+		$menu    = $args['menu'];
+		$id      = $args['id'];
+		$options = get_option( $menu );
+		$token   = '';
+
+		if ( isset( $options['np_relay_server_url'] ) ) {
+			$url = $options['np_relay_server_url'];
+		} else {
+			$url                       = '';
```

Comment:
The = needs to be indented correctly. Here and above for the same variable on line 594.